Apps/phhackerone/hackerone.json

```diff
@@ -0,0 +1,653 @@
+{
+    "appid": "2ab1b33f-1951-4dbd-862b-8347e511a6d5",
+    "name": "HackerOne",
+    "description": "This app integrates with HackerOne to support various generic and investigative actions",
+    "publisher": "Splunk Community",
+    "type": "generic",
+    "main_module": "hackerone_connector.pyc",
+    "app_version": "1.5.0",
+    "utctime_updated": "2020-01-28T00:00:00.000000Z",
+    "package_name": "phantom_hackerone",
+    "product_vendor": "HackerOne",
+    "product_name": "HackerOne",
+    "product_version_regex": ".*",
+    "min_phantom_version": "4.8.24304",
+    "logo": "logo_hackerone.svg",
+    "logo_dark": "logo_hackerone_dark.svg",
+    "license": "Copyright (c) 2020 Splunk Inc.",
+    "configuration": {
+        "api_identifier": {
+            "description": "HackerOne Identifier",
+            "data_type": "string",
+            "order": 0,
+            "required": true
+        },
+        "api_token": {
+            "description": "HackerOne API Token",
+            "data_type": "password",
+            "order": 1,
+            "required": true
+        },
+        "program_name": {
+            "description": "HackerOne Program Name",
+            "data_type": "string",
+            "order": 2,
+            "required": true
+        },
+        "state_filter": {
+            "description": "Filter for report state (to be used when polling)",
+            "data_type": "string",
+            "order": 3
+        },
+        "assignment_filter": {
+            "description": "Filter for report assignment (to be used when polling)",
+            "data_type": "string",
+            "order": 4
+        },
+        "full_comments": {
+            "description": "Collect comments for reports (to be used when polling, much slower)",
+            "data_type": "boolean",
+            "order": 5
+        },
+        "phantom_api_token": {
+            "description": "Phantom API token for updating existing containers",
+            "data_type": "password",
+            "order": 6,
+            "required": true
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied credentials",
+            "type": "test",
+            "identifier": "test_asset_connectivity",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get report",
+            "description": "ID of report to get the report",
+            "type": "investigate",
+            "identifier": "get_report",
+            "read_only": true,
+            "parameters": {
+                "report_id": {
+                    "description": "ID of report to update",
```

Comment:
The action for this parameter is named `get report` and is marked as `read_only`, so should this be something like: `ID of report to get` or  just `ID of report`?

Reply:
Fixed